### PR TITLE
Feature/resetandlevel

### PR DESCRIPTION
**Pull request by albert-smit**

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/semgr8/logger/LogLevel.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
      *
      * @param label the string used to display this enum.
      */
-    private LogLevel(String label) {
+    LogLevel(String label) {
```
Comment (albert-smit) [inline src/main/java/nl/tudelft/ti2206/semgr8/logger/LogLevel.java]: why was this constructors visibility changed?

File: src/main/java/nl/tudelft/ti2206/semgr8/model/LineRemover.java [hunk 2]
```diff
@@ -0,0 +1,598 @@
+package nl.tudelft.ti2206.semgr8.model;
```
Comment (albert-smit) [inline src/main/java/nl/tudelft/ti2206/semgr8/model/LineRemover.java]: this file was supposed to be gone right?

File: src/test/java/nl/tudelft/ti2206/semgr8/controller/GameControllerTest.java [hunk 3]
```diff
@@ -51,10 +51,12 @@
...
         // return a mocked property
         when(game.getScoreProperty()).thenReturn(propertyMock);
+        when(game.getLevelProperty()).thenReturn(propertyMock);
+        when(game.getComboProperty()).thenReturn(propertyMock);
 
         gameController = new GameController(guiController, game);
-        when(game.getBoard()).thenReturn(board);
-        when(e1.getPosition()).thenReturn(p1);
+
+        when(e1.getPosition()).thenReturn(p1);when(game.getBoard()).thenReturn(board);
```
Comment (albert-smit) [inline src/test/java/nl/tudelft/ti2206/semgr8/controller/GameControllerTest.java]: missing newline

File: src/test/java/nl/tudelft/ti2206/semgr8/model/BoardTest.java [hunk 4]
```diff
@@ -280,6 +294,42 @@ public void testHashCodeBoard() {
         assertEquals(c.hashCode(), d.hashCode());
     }
 
+//    @Test
```
Comment (albert-smit) [inline src/test/java/nl/tudelft/ti2206/semgr8/model/BoardTest.java]: commented out code

File: src/main/java/nl/tudelft/ti2206/semgr8/logger/LogLevel.java (hunk 1 above)
Comment (LeonWubben) [inline src/main/java/nl/tudelft/ti2206/semgr8/logger/LogLevel.java]: According to checkstyle the private modifier is redundant for enum constructors
image

**Comment by albert-smit** [comment]
because of time, i have fixed these comments myself, and will also merge it

**Pull request merged**